8.8/vips.modules

```diff
@@ -951,7 +957,7 @@
       module="v8.8.0-rc1/vips-8.8.0-rc1a.tar.gz"
       version="8.8.0"
       checkoutdir="vips-8.8.0">
-      <patch file="file:///home/john/develop/transform-7.32/transform.patch" strip="1"/>
+      <patch file="patches/transform.patch" strip="1"/>
```

Comment:
Is the `transform.patch` intentionally committed to version control? I could remember an old issue:
https://github.com/libvips/build-win32/issues/2#issuecomment-8796626

Reply:
It's 25 years now and he's retured, so I think he's no longer worried about this.